kbchat/kvstore_test.go

```diff
@@ -0,0 +1,102 @@
+package kbchat
+
+import (
+	"testing"
+
+	"github.com/keybase/go-keybase-chat-bot/kbchat/types/keybase1"
+	"github.com/stretchr/testify/require"
+)
+
+func contains(a []string, x string) bool {
+	for _, n := range a {
+		if x == n {
+			return true
+		}
+	}
+	return false
+}
+
+func containsKey(a []keybase1.KVListEntryKey, x string) bool {
+	for _, n := range a {
+		if x == n.EntryKey {
+			return true
+		}
+	}
+	return false
+}
+
+// test doesn't make assumption about prior state
+func TestKVStore(t *testing.T) {
+	var err error
+
+	alice, dir := testBotSetup(t, "alice")
+	channel := getTeamChatChannel(t, "acme")
+	team := channel.Name
+	defer testBotTeardown(t, alice, dir)
+
+	namespace := "_test_namespace1"
+	key := "_test_key1"
```

Comment:
nit: can we call this `entryKey`? i was pretty intentional with naming in other places to use `entryKey` instead of just `key`, since "keys" are also things we use for encryption and signing.